internal/proxy/plan_parser.go

```diff
@@ -237,6 +237,10 @@ func getCompareOpType(opStr string, reverse bool) (op planpb.OpType) {
 		op = planpb.OpType_Equal
 	case "!=":
 		op = planpb.OpType_NotEqual
+	case "startsWith":
```

Comment:
can use '^' for "begin from", and '$' for "end with" ?

Reply:
We can simply support the basic expression in this patch and then extend the grammar rules of expression later.
For now, `startsWith` and `endsWith` are the default supported version in our parser library.